themes.md

```diff
@@ -14,7 +14,7 @@ Linux is a kernel, and has nothing to do with themes. While some desktop environ
 
 - [FreeDesktop.org](https://freedesktop.org) specifications try to ensure different desktops' _basic features_ are inter-compatible, but does not address visual style.
 
-  - There are specs for _icon_ and _sound_ themes, but even these are both incomplete and outdated.
+  - There are specs for _icon_ and _sound_ themes, but some desktops do not support these.
```

Comment:
I recall the issue with from working on elementary being that they both didn't cover the actual range of icons and sounds you'd want to use, and seemed to be outdated in what they did dictate. However, this is the Icon Naming spec and not the Icon Theme spec, so perhaps its moot. I think this slight rewording remains accurate, anyway:

```suggestion
  - There are specs for _icon_ and _sound_ themes, but some platforms do not support them.
```